client-libraries/java/rest-client/src/com/google/android/gcm/server/Constants.java

```diff
@@ -63,6 +63,11 @@
   public static final String PARAM_TIME_TO_LIVE = "time_to_live";
 
   /**
+   * Prefix to HTTP parameter used to set the message priority.
```

Comment:
This is not a prefix, this is the parameter name. Please change the docs for time to live as well.